src/main/java/com/nvidia/spark/rapids/jni/GpuTimeZoneDB.java

```diff
@@ -27,37 +28,39 @@
 import java.util.Map;
 import java.util.TimeZone;
 import java.util.concurrent.*;
+import java.util.function.Function;
 
-import ai.rapids.cudf.ColumnVector;
-import ai.rapids.cudf.DType;
-import ai.rapids.cudf.HostColumnVector;
-import ai.rapids.cudf.Table;
+import ai.rapids.cudf.*;
 
 public class GpuTimeZoneDB {
 
   public static final int TIMEOUT_SECS = 300;
 
-
   // For the timezone database, we store the transitions in a ColumnVector that is a list of 
   // structs. The type of this column vector is:
-  //   LIST<STRUCT<utcInstant: int64, localInstant: int64, offset: int32>>
+  //   LIST<STRUCT<utcInstant: int64, localInstant: int64, offset: int32, looseInstant: int64>>
   private CompletableFuture<Map<String, Integer>> zoneIdToTableFuture;
   private CompletableFuture<HostColumnVector> fixedTransitionsFuture;
+  private CompletableFuture<HostColumnVector> zoneIdVectorFuture;
+  // Used to store Java ZoneId.SHORT_IDS Map, e.g.: PST:America/Los_Angeles
+  // Note: also add a entry: Z->UTC
+  private HostColumnVector shortIDs;
```

Comment:
Thanks for the comments. Why is this not a `CompletableFuture` like the others? Is it tied to one of the others? Should we just have a singe future for is everything loaded?

Also if we already have a mapping from ID to location, why don't we include the SHORT IDs in it too?

Reply:
GpuTimeZoneDB.java has memory leak.
Refer to https://github.com/NVIDIA/spark-rapids-jni/pull/1689.
Will rebase GpuTimeZoneDB.